### PR TITLE
Label Unit Tests

### PR DESCRIPTION
**Pull request by tomjackking**
- added TS tests for label
- added new tests for previous components I worked on to increase coverage
- minor change to nesting-test and overrides-test to export to avoid TS issues
- minor changes to WBC types to add key to colors to avoid TS issues in test

### CODE REVIEW

File: brands/WBC/src/types.d.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ declare module '@westpac/wbc' {
 		code: string;
 	};
 	type Colors = {
+		[key: string]: string;
```
Comment (samithaf) [inline brands/WBC/src/types.d.ts:7]: Colours needs to pre-defined set of key-pair values. So please remove this

Comment (tomjackking) [reply brands/WBC/src/types.d.ts]: This line has been removed

File: components/form/tests/unit/fieldset.spec.js [hunk 2]
```diff
@@ -41,6 +41,14 @@ describe('Fieldset component', () => {
 		expect(getByText(/test hint/).getAttribute('class')).toMatch(/form-hint/gi);
 	});
 
+	test('should render Fieldset component when hint prop contains function', () => {
+		const handleClick = jest.fn(() => {});
+		const { getByTestId } = render(
+			<SimpleFieldset hint={handleClick} data-testid="test-fieldset" />
```
Comment (wove5) [inline components/form/tests/unit/fieldset.spec.js:47]: Where does data-testid get used?

File: components/label/tests/unit/label.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+nestingTest({
+	name: 'label',
+	Component: (props: any) => <Label look="primary" value="default" {...props}></Label>,
+});
+
+function SimpleLabel(props: any) {
+	return (
+		<GEL brand={wbc}>
+			<Label look={props.look} {...props} data-testid="test-label" />
```
Comment (wove5) [inline components/label/tests/unit/label.spec.tsx:23]: Where does data-testid get used?

**Review by samithaf — APPROVED** [review]

**Pull request merged**